spec/models/doorkeeper/access_token_spec.rb

```diff
@@ -143,6 +143,12 @@ module NoGenerate
         subject.resource_owner_id = nil
         expect(subject).to be_valid
       end
+
+      it 'is valid without resource_owner_id' do
```

Comment:
Oops, copy-pasta issues here. Sorry, seems I forgot to edit this `s/resource_owner_id/application_id`